### PR TITLE
Change subscan request to subquery

### PR DESCRIPTION
**Pull request by FrogTravel**

### CODE REVIEW

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/SubscanValidatorSetFetcher.kt [hunk 1]
```diff
@@ -53,139 +38,25 @@ internal class CallDescription(
...
-    }
-
-    private fun extractNominationsFromArgument(argument: Any?): List<String> {
-        require(argument is List<*>?)
+        )
 
-        return argument.orEmpty().mapNotNull(::extractAccountIdFromArgument)
+        return validatorsInfos.data.query?.eraValidatorInfos?.nodes?.map {
+            it.address
+        }?.distinct() ?: emptyList()
```
Comment (valentunn) [inline feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/SubscanValidatorSetFetcher.kt:60]: distinct().orEmpty()

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/SubscanValidatorSetFetcher.kt [hunk 2]
```diff
@@ -53,139 +38,25 @@ internal class CallDescription(
...
-        else -> null
-    }
-
-    private fun extractNominationsFromArgument(argument: Any?): List<String> {
-        require(argument is List<*>?)
+        )
 
-        return argument.orEmpty().mapNotNull(::extractAccountIdFromArgument)
+        return validatorsInfos.data.query?.eraValidatorInfos?.nodes?.map {
+            it.address
```
Comment (valentunn) [inline feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/SubscanValidatorSetFetcher.kt:59]: map(Node::address)

**Review by valentunn — APPROVED** [review]

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/SubscanValidatorSetFetcher.kt (hunk 1 above)
Comment (FrogTravel) [reply feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/SubscanValidatorSetFetcher.kt]: Done

File: feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/SubscanValidatorSetFetcher.kt (hunk 2 above)
Comment (FrogTravel) [reply feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/SubscanValidatorSetFetcher.kt]: Done

**Pull request merged**